includes/EditPage.php

```diff
@@ -2081,7 +2081,7 @@ protected function showHeader() {
 					if ( !$revision->isCurrent() ) {
 						/* Wikia change begin - @author: Marooned */
 						/* Add new hook and condition to allow alternate message to be displayed when editing old revision */
-						if ( wfRunHooks( 'EditPage::showEditForm:oldRevisionNotice', array( &$this ) ) ) {
+						if ( wfRunHooks( 'EditPage::showEditForm:oldRevisionNotice', array( $this ) ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)